inc/osvr/Util/ProcessUtils.h

```diff
@@ -0,0 +1,91 @@
+/** @file
+@brief Header
```

Comment:
I try to put useful file comments for files in Util - I don't always succeed, but I try.